src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Light clients
+description: Introduction to Ethereum light clients.
+lang: en
+---
+
+Running a full node is the most trustless, private, decentralized and censorship resistant way to interact with Ethereum. With a full node you keep your own copy of the blockchain that you can query instantly and you get direct access to Ethereum's peer-to-peer-network. However, running a full node requires a substantial amount of memory, storage and CPU. This can be expensive, meaning it is not feasible for everyone to run their own node. There are several solutions to this on the Ethereum roadmap, including statelessness, but they are several years away from being implemented. The answer in the near-term is to trade-off some of the benefits of running a full node for large performance improvements that allow nodes to run with very low hardware requirements. Nodes that make this trade-off are known as light nodes.
+
+## What is a light client {#what-is-a-light-client}
+
+A light node is a node running light client software. Instead of keeping local copies of the blockchain data and independently verifying all the changes, they request the necessary data from some provider instead. The provider might be a full node or a centralized RPC server. The data is then verified by the light node, allowing it to keep up with the head of the chain. The light node only processes block headers, only occasionally downloading the actual block contents. Nodes can vary in their lightness, depending upon whether the combinations of light and full client software they run. For example, the lightest configuration would be to run a light execution client and a light consensus client. It is also likely that many nodes will choose to run light consensus clients with full execution clients, or vice-versa.
```

Comment:
Might suggest expanding the topic sentence here a little, perhaps something like:

> A light node is a device running light client software, designed to enable verification of the chain with minimal resource requirements.

---

> The provider might be a full node or a centralized RPC server.

Wouldn't they all be full nodes, even as a centralized RPC server?

Reply:
(No hyphen in vice versa)

Reply:
```suggestion
A light node is a node running light client software. Instead of keeping local copies of the blockchain data and independently verifying all the changes, they request the necessary data from some provider instead. The provider might be a direct connection to a full node or via some centralized RPC server. The data is then verified by the light node, allowing it to keep up with the head of the chain. The light node only processes block headers, only occasionally downloading the actual block contents. Nodes can vary in their lightness, depending upon whether the combinations of light and full client software they run. For example, the lightest configuration would be to run a light execution client and a light consensus client. It is also likely that many nodes will choose to run light consensus clients with full execution clients, or vice versa.
```